.codecov.yaml

```diff
@@ -0,0 +1,11 @@
+coverage:
+  # Commit status https://docs.codecov.io/docs/commit-status are used
+  # to block PR based on coverage threshold.
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 0%
```

Comment:
Since this is a new repo, we can start out with "ideal" thresholds.  I'm wondering if 0% is too aggressive
for Golang. If you add more error cases to a file the coverage can drop slightly (which I think is OK). But IMO, the overall coverage should never drop below 80%. In my exp, this has been a pretty good policy for Go. Thoughts?

Reply:
LGTM. We can always use admin override when this is violated.

Reply:
And we can always adjust the policy later.

Reply:
Still need update thresholds? I can do it. @artursouza @pkedy